app/views/overview/_service-instance-row.html

```diff
@@ -124,3 +127,9 @@
   </div>
 
 </div>
+<overlay-panel single-column="true" show-panel="row.overlay.panelVisible" show-close="true" handle-close="row.closeOverlayPanel">
+  <div ng-if="row.overlay.panelName === 'bindService'">
```

Comment:
i wont be shocked if we have more in the future, so just set this up the same way i was doing in the Add to Project dropdown PR